<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QtabWidget{
border-radius:30px;
border-color:rgb(0, 0, 0)
}
*{
font-family:century gothic;
font-size:17px;
}
QLineEdit
{
border-radius:11px;
border-color:rgb(0, 0, 0);

}
QPushButton
{
background:rgb(139, 4, 35);
border-radius:13px;
color:rgb(255, 255, 255);
}
QPushButton:hover
{
background:rgb(216, 5, 37);
border-radius:13px;
color:rgb(255, 255, 255);
}

QToolButton{
background:rgb(139, 4, 35);
border-radius:15px;
}
QToolButton:hover{
background:rgb(216, 5, 37);
border-radius:15px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>551</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="stackedWidgetPage1">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ID Equipement</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Type </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_equipement">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>61</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_type_equipement">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>111</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prix">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>161</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nombre">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>211</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter_equipement">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>310</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="ajouter_equipements">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="affiche_equipements">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier_equipements">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer_equipements">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>280</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage2">
     <widget class="QToolButton" name="toolButton_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/[removal.ai]_tmp-621f8f69f3ee1.png</normaloff>../../Downloads/[removal.ai]_tmp-621f8f69f3ee1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string> Recherche...</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>491</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/[removal.ai]_tmp-621f91100d62d.png</normaloff>../../Downloads/[removal.ai]_tmp-621f91100d62d.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/transparent-sort-icon-interface-icon-assets-icon-arrows-icon-5f8d15e9d61166.7886260416030817058768-removebg-preview.png</normaloff>../../Downloads/transparent-sort-icon-interface-icon-assets-icon-arrows-icon-5f8d15e9d61166.7886260416030817058768-removebg-preview.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <zorder>lineEdit_12</zorder>
     <zorder>toolButton_2</zorder>
     <zorder>tableView</zorder>
     <zorder>toolButton_3</zorder>
     <zorder>toolButton_4</zorder>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage3">
     <widget class="QPushButton" name="modifier_equipement">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>310</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_type_modif">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>111</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ID Equipement</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_equi_modif">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>61</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nombre_modif">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>211</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Type </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prix_modif">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>161</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage4">
     <widget class="QPushButton" name="supprimer_equipement">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ID Equipement</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_supprimer_equipement">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>131</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>130</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/[removal.ai]_tmp-621f8f69f3ee1.png</normaloff>../../Downloads/[removal.ai]_tmp-621f8f69f3ee1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QToolButton" name="return_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>510</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/seta-small-direita-branca-300x258.png</normaloff>../../Downloads/seta-small-direita-branca-300x258.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>27</width>
      <height>27</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
